packages/waku/src/router/fs-router.ts

```diff
@@ -93,6 +94,16 @@ export function unstable_fsRouter(
           .replace(/\.\w+$/, '')
           .split('/')
           .filter(Boolean);
+        // must come before isIgnoredPath check to include slices from inside _components
+        if (pathItems.at(-1)?.startsWith('_slice')) {
+          createSlice({
+            component: mod.default,
+            render: 'dynamic',
+            id: pathItems.at(-1)!.slice('_slice'.length),
```

Comment:
Another idea is to keep `slice`? Not sure which is more ituitive.
```suggestion
            id: pathItems.at(-1)!.slice('_'.length),
```

Reply:
I think that would make the id's read better. i'll change to that 👍